apps/web/cypress/tests/integration-store.spec.ts

```diff
@@ -0,0 +1,78 @@
+import { ChannelTypeEnum } from '@notifire/shared';
+
+describe('Integration store page', function () {
+  beforeEach(function () {
+    // @ts-ignore
+    cy.initializeSession().as('session');
+  });
+
+  it('should display email available for connection', function () {
+    cy.visit('/integrations');
+    cy.location('pathname').should('equal', '/integrations');
+
+    getFirstIntegrationCard().get('button').contains('Connect');
+  });
+
+  it('should display integrated sendgrid provider', function () {
+    interceptIntegration(true);
+
+    cy.visit('/integrations');
+
+    getFirstIntegrationCard().getByTestId('card-status-bar-active').contains('Active');
+
+    getFirstIntegrationCard()
+      .getByTestId('card-status-bar-active')
+      .should(($div) => {
+        const text = $div.text();
+
+        expect(text).to.match(/^Active$/);
+      });
+  });
+
+  it('should display not integrated sendgrid provider', function () {
+    interceptIntegration(false);
+
+    cy.visit('/integrations');
+
+    getFirstIntegrationCard().getByTestId('card-status-bar-active').contains('Active');
+
+    getFirstIntegrationCard()
+      .getByTestId('card-status-bar-active')
+      .should(($div) => {
+        const text = $div.text();
+
+        expect(text).to.match(/^Not Active$/);
+      });
```

Comment:
I think, can be done with contains to simplify it
```suggestion
      .contains('Not Active');
```